### PR TITLE
Read me game instructions

### PR DESCRIPTION
**Pull request by imhilla**
In this milestone, we had the chance to explain the TicTacToe game.
The first section introduces how the game is played
The second section explains the game rules.
The lower section explains how to set up locally and play on the command line

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,35 @@
...
+
+The first player that has a line of three X's or three O's wins the game.
+
+## How to set up in local machine
+
+Tictactoe is easier to setup. You can use Sourcetree, Git from the command line, or any client you like to clone your Git repository. These instructions show you how to clone your repository using Git from the terminal. First, from the repository, click + in the global sidebar and select Clone this repository under Get to work. Secondly, Copy the clone command (either the SSH format or the HTTPS). If you are using the SSH protocol, ensure your public key is in github and loaded on the local system to which you are cloning. Forth, From a terminal window, change to the local directory where you want to clone your repository. Paste the command you copied from Github.
+
+If the clone was successful, a new sub-directory appears on your local drive in the directory where you cloned your repository. This directory has the same name as the github repository that you cloned. The clone contains the files and metadata that Git requires to maintain the changes you make to the source files. At that point you will be ready to view enumerable files and install them locally.
```
Comment (whiz25) [inline README.md]: - how does someone clone the repo. what command should they type? include technical details
- be precise with the instructions

File: README.md [hunk 2]
```diff
@@ -1,2 +1,35 @@
...
-A game built with Ruby
+## Tic-Tac-Toe
+
+In this project we had the opportunity to build Tic Tac Toe game in Ruby. Just in case you're unfamiliar with the game, let's discuss the rules first. Tic Tac Toe is played on a 3 x 3 grid of available cells, and the goal of the game is to place three of your tokens (X or O) on the grid in a row, including diagonals.
+
+Tic Tac Toe is a game that flexes the basic concepts in programming. For example, a Tic Tac Toe program requires data structures for storing the board and conditional logic for knowing whose turn it is or if someone has won.
+
+It is a game played by two people, each trying to make a line of three X's or three O's in a boxlike figure with nine spaces. It is either won horizontally, vertically or diagonally.
+
+The first player that has a line of three X's or three O's wins the game.
```
Comment (whiz25) [inline README.md]: - divide this into two sections. move some content from here to a "Rules section" for the game.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,35 @@
...
+Tictactoe is easier to setup. You can use Sourcetree, Git from the command line, or any client you like to clone your Git repository. These instructions show you how to clone your repository using Git from the terminal. First, from the repository, click + in the global sidebar and select Clone this repository under Get to work. Secondly, Copy the clone command (either the SSH format or the HTTPS). If you are using the SSH protocol, ensure your public key is in github and loaded on the local system to which you are cloning. Forth, From a terminal window, change to the local directory where you want to clone your repository. Paste the command you copied from Github.
+
+If the clone was successful, a new sub-directory appears on your local drive in the directory where you cloned your repository. This directory has the same name as the github repository that you cloned. The clone contains the files and metadata that Git requires to maintain the changes you make to the source files. At that point you will be ready to view enumerable files and install them locally.
+
+## Authors
+
+👤 Taiwo Coker
+
+Github: @taiwocoker
+Twitter: @SelloCoker
```
Comment (whiz25) [inline README.md]: - add a section for contributing. be detailed how they will contribute.

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Awesome progress :+1: 
Your readme looks good but it's not professional enough. Make the changes I have suggested and make this more professional.

### Martin Afani

**Review by pelzolga123 — APPROVED** [review]
Approved.

**Comment by pelzolga123** [comment]
## Great work, congratulation :+1: Project Approved.
You did all changes required previously, but I would still advise a few more changes, to make README file look better:
* Make all headers the same size, except for `Tic-Tac-Toe`, it should be bigger than other headers.
* The header of `contributing` section does not look like header, please kindly change that.

Then go ahead and merge to develop.
_Goodluck in your next project_ *Happy coding*

**Comment by imhilla** [comment]
> ## Great work, congratulation 👍 Project Approved.
> You did all changes required previously, but I would still advise a few more changes, to make README file look better:
> 
> * Make all headers the same size, except for `Tic-Tac-Toe`, it should be bigger than other headers.
> * The header of `contributing` section does not look like header, please kindly change that.
> 
> Then go ahead and merge to develop.
> _Goodluck in your next project_ _Happy coding_

noted and changed

**Pull request merged**